gr-digital/lib/framer_sink_1_impl.cc

```diff
@@ -80,14 +79,14 @@ int framer_sink_1_impl::work(int noutput_items,
     int count = 0;
 
     if (VERBOSE)
-        fprintf(stderr, ">>> Entering state machine\n");
+        GR_LOG_INFO(d_debug_logger, "enter state machine");
 
     while (count < noutput_items) {
         switch (d_state) {
 
         case STATE_SYNC_SEARCH: // Look for flag indicating beginning of pkt
             if (VERBOSE)
-                fprintf(stderr, "SYNC Search, noutput=%d\n", noutput_items);
+                GR_LOG_INFO(d_debug_logger, "SYNC Search, noutput=%d");
```

Comment:
oops, `noutput_items` is not printed anymore now!

Reply:
whew, thanks for catching that!